### PR TITLE
Refactor wireframe indices to `WireframePipelineStage`

### PR DESCRIPTION
**Pull request by ptrgags**
This PR refactors the generation of wireframe indices to be its own stage in the `ModelExperimental`. This way, `buildDrawCommands()` has less complexity.

Along the way I fixed a couple of minor issues:

* `ModelExperimentalPrimitive` and `ModelExperimentalNode` were configuring the pipeline in the constructor. This really isn't necessary.
* I noticed some tests in `ModelExperimental` were incorrectly requesting a WebGL 2 context. It was easily fixed by changing the request to `createScene({contextOptions: {requestWebgl2: true}})`

[Local sandcastle](http://localhost:8080/Apps/Sandcastle/index.html#c=bVPva9swEP1XRD45kEopaRlkaVmXDBZY19KGlYG/yNLFOSZLRjrnR8f+90m2U5Ktn6w7vffu7p0sBPvcoNFoS6YlSaZc4wnCgbk1+/5zzh5qsIt0UTtP0kzZhqieCrHb7S65PSheuq0ISCC0QyKBFgkl4RaCmOiLohfntSwht8rZQGyLsAPPbpiFHZtDwKbiP9pclg9UG8+dJYkWfD4Ysd+5ZYzA+5h59G6LGvz0SFQeJMGL80avOkg2HOX2z/BjbnPbleJBgQVeGlcA11DTZgWB7soIDtSzYjvkGzhlwZ7A6qwv1CW7YLJYoYGwtKEGRc7f4x5tqtgNiMd8mureaTBRvRdV7wq06i2ybaCToQQBOjfqrAGgrDWn8ebNkKWzTxDiGhXwtXfVXYiwpc4+XE+uJskaxsDKwkBb7su+Bo8V2Ha5yYETxAKKpnxBD2svK+iue2vPjK2jArY751LrrO+720Af8LgmfYjbqzAApw3YbN1YRegsy4bdinvNV+eqlctShh09GHXRiQ9fo158WI9IavMkbQk9gbExH4+O54sxv34Ljq0UrrGJ+1xvwAP3UagJTLArPu6gw/RJ7bejDkaDWaCDgduj0Ces0u+QbM84FwRVbeIjDKJo1K9YQIWQiAk6E6fUmcYtQ33zzjNnysgQ4s26MeYZXyEf3M5ExP9HNa6d/GEL3shDgm0ub791Sc75TMTwfSY5Zwrp/1H+Cw)

@j9liu could you review when you get a chance?

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @ptrgags!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/wireframe-stage/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

File: Source/Scene/ModelExperimental/buildDrawCommands.js [hunk 1]
```diff
@@ -182,9 +160,10 @@ function deriveTranslucentCommand(command) {
   return derivedCommand;
 }
 
-function getIndexBuffer(primitiveRenderResources, debugWireframe, frameState) {
-  if (debugWireframe) {
-    return createWireframeIndexBuffer(primitiveRenderResources, frameState);
+function getIndexBuffer(primitiveRenderResources, frameState) {
```
Comment (j9liu) [inline Source/Scene/ModelExperimental/buildDrawCommands.js:163]: does this need a `/** @private */` comment on top of it? looks like I forgot to add one in my PR

**Review by j9liu — COMMENTED** [review]
@ptrgags looks good overall, just left a few comments regarding documentation

File: Specs/Scene/ModelExperimental/WireframePipelineStageSpec.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        primitiveType: primitive.primitiveType,
+        wireframeIndexBuffer: undefined,
+        model: {
+          _resources: resources,
+        },
+      };
+    }
+
+    it("Creates wireframe indices from buffer (WebGL 2)", function () {
+      return loadGltf(boxTexturedBinary, sceneWithWebgl2).then(function (
```
Comment (j9liu) [inline Specs/Scene/ModelExperimental/WireframePipelineStageSpec.js:85]: This unit test needs to return if the scene wasn't able to request a webgl2 context

**Comment by ptrgags** [comment]
@j9liu updated!

**Comment by j9liu** [comment]
@ptrgags merging, thanks!

**Pull request merged**